manifests/params.pp

```diff
@@ -278,6 +278,7 @@
     $agent_logfile                          = '/var/log/zabbix/zabbix_agentd.log'
     $agent_logfilesize                      = '100'
   }
+  $agent_jmx_port                           = 0
```

Comment:
Should the default be undef? In that case, the datatype should be `Optional[Stdlib::Port]`

Reply:
I use the check on 0 later on in the code. So it should be 0

Reply:
Please check for nil/undef. Checking for 0 seems kind like a hack :)